### PR TITLE
ScentreGroup case study

### PR DESCRIPTION
**Pull request by joshprice**

### CODE REVIEW

File: src/content/case-studies/scentre-group.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+increasing customer intimacy and accelerating agility and innovation.
+
+As part of these goals, Scentre engaged Alembic to create a platform to sell Digital Out of Home advertising campaigns.
+Westfield shopping centres have a network of almost 2000 smart screens delivering premium ads to discerning customers. We helped 
+design and build the tool which enables the sales team to work better and faster using highly targeted screen selection based on 
+the campaign goals.
+
+## Section 2: The Solution
+
+Alembic worked closely with CXT teams to research and architect the platform to process large amounts
```
Comment (freshtonic) [inline src/content/case-studies/scentre-group.mdx]: "with the"

Please also expand what CXT stands for (Customer Experience Team I think?) at the first usage of the acronym. You can keep using the acronym after that.

File: src/content/case-studies/scentre-group.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+of data provided by different systems. One of the goals of this project was to provide a centralised platform for
+the sales team as well as deliver a maintainable and documented system for CXT engineers to manage after Alembic's engagement.
+
+The team designed a platform powered by a front-end in React and TypeScript, consuming data from  
+a GraphQL API in Node.js, developed also in TypeScript. The choice of using Typescript was one of the most important decisions
+because it simplifies the JavaScript code adding types, making it easier to read and debug. This is trully important for a platform that has to be maintainable over time.
+
+The most complex part of this project lives in the GraphQL API where the system has to work with all the data related to the
+Westfield centres and the super screens. Over this data, the filters are applied in order to suit the sales team
+requirements and target the best screen to show the clients' campaigns. Most of the data comes from an enterprise
```
Comment (freshtonic) [inline src/content/case-studies/scentre-group.mdx]: "most effective screens"

File: src/content/case-studies/scentre-group.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+The first end-to-end version of the application was created in 3 months, where we deployed into staging environments
+and connected to the data APIs to collect the information related to the screens. The entire platform took
+less than 7 months to be production ready and used by the sales team to process the advertising campaigns. 
+
+## Section 3: The Outcomes
+
+Alembic delivered a platform for Scentre Group's sales teams to be able to filter and select smart screens and retail sites where Westfield companies run their advertising campaigns.  Alembic worked directly with the Customer Technology Experience team on site as well
+as remotely to deliver the project. We had daily stand-ups and fortnightly iterations so the team could receive feedback from all the stakeholders and adapt the system while in development. 
+
+The application had to get large amounts of data from a Data Warehouse continuously to have the most
```
Comment (freshtonic) [inline src/content/case-studies/scentre-group.mdx]: This paragraph can be removed - I don't think it's necessary.

**Comment by freshtonic** [comment]
@sechosebio thanks for this José!  Can you address my feedback and I'll get it merged. Cheers!

File: src/content/case-studies/scentre-group.mdx (hunk 1 above)
Comment (sechosebio) [reply src/content/case-studies/scentre-group.mdx]: @freshtonic  The meaning of CXT is in the line [16](https://github.com/team-alembic/alembic-website/pull/36/files#diff-c48430081b449ed6ff8fba0995fdb99aR16). Maybe is better to write it in the other way around?

File: src/content/case-studies/scentre-group.mdx (hunk 3 above)
Comment (sechosebio) [reply src/content/case-studies/scentre-group.mdx]: From  `The application` to the `data aggregation..` ?

**Pull request merged**